### DISCUSSION TITLE
Does this replace the "old" github.community?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [community/community#28647] [Discussions]
A few years ago, GitHub created https://github.community

That URL now redirects to https://github.com/orgs/community/discussions/

Does this "new" community replace the "old" community (which was just a few years old)?

Is the content from the "old" community available somewhere?  I had asked a couple questions, and received helpful responses; I would like to review those answers.

**Comment by jfullstackdev** [comment]
yes this is the new GitHub Community, under one roof, because it's already here at GitHub

as for those old contents, just those with marked answers were transferred,

**Reply by Gitoffthelawn** [reply]
Thanks for your response... I appreciate it.

I just sent the below to Melvin, but if you get a chance first, that works too...

When I perform a search, I'm not seeing anything (besides this discussion) with my username, so it sounds like all that good info was lost when GitHub closed down the former community site. Would you mind trying a search to see if the results you obtain are any different?

**Reply by NitinIsFit** [reply]
Then i guess you should mark his answer as "answered" 😂🤪

**Comment by melvincwng** [comment]
@Gitoffthelawn, the old GitHub.community forums have been removed already.

Moving forward, this platform here Github Discussions will be the new place for existing members of GitHub to share their thoughts, ask questions or provide feedback etc.

Hope this clarifies.

**Reply by Gitoffthelawn** [reply]
Thanks Mel

> @Gitoffthelawn, the old GitHub.community forums have been removed already.
> 
> Moving forward, this platform here Github Discussions will be the new place for existing members of GitHub to share their thoughts, ask questions or provide feedback etc.
> 
> Hope this clarifies.

Thanks Melvin.  I appreciate your response.

When I perform a search, I'm not seeing anything (besides this discussion) with my username, so it sounds like all that good info was lost when GitHub closed down the former community site.   Would you mind trying a search to see if the results you obtain are any different?

**Comment by NitinIsFit** [comment]
*gulps* the GitHub.community link redirected me here again.

**Reply by Gitoffthelawn** [reply]
At least we can finally stop using that URL.  It was an auto-complete mess given that ".community" begins with ".com".

**Comment by jfullstackdev** [comment]
you may want to further read the thread here 

https://github.com/orgs/community/discussions/27346
